test/terraform/aws-persistent/main.tf

```diff
@@ -16,20 +16,81 @@ resource "random_password" "db_password" {
   special = false
 }
 
+variable "operator_version" {
+  type    = string
+  default = "v25.2.0-beta.1"
+}
+
+variable "orchestratord_version" {
+  type    = string
+  default = "v0.130.3"
+}
+
 module "materialize_infrastructure" {
-  source = "git::https://github.com/MaterializeInc/terraform-aws-materialize.git?ref=v0.2.0"
+  source = "git::https://github.com/MaterializeInc/terraform-aws-materialize.git?ref=v0.2.5"
 
   # Basic settings
   namespace    = "aws-persistent"
   environment  = "dev"
   install_materialize_operator = true
+  use_local_chart = true
+  helm_chart = "materialize-operator-v25.2.0-beta.1.tgz"
+  operator_version = var.operator_version
+  orchestratord_version = var.orchestratord_version
 
   helm_values = {
-      defaultReplicationFactor = {
-          system = 1
-          probe = 1
-          support = 1
-          analytics = 1
+      clusters = {
```

Comment:
The `clusters` section is under the `operator` key in the values file:

https://github.com/MaterializeInc/materialize/blob/main/misc/helm-charts/operator/values.yaml#L11

Reply:
So is it true that if I set anything in `operator` I have to override everything?

Reply:
No, this should not be the case. It will override only the values that you are modifying:

> https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#overriding-values-from-a-parent-chart

Are you seeing a different behavior?

Reply:
I didn't get it into a working state, I'm not sure what I'm doing wrong with the helm_values or if they don't work. I just manually alter the cluster now and that works for me. So can be merged in this state.